packages/sdks-tests/src/e2e-tests/editing.spec.ts

```diff
@@ -369,4 +370,35 @@ test.describe('Visual Editing', () => {
       expect(consoleMsg).toContain('apiKey: abcd');
     });
   });
+
+  test.describe('Content Input', () => {
+    test('correctly updates', async ({ page, packageName, basePort, sdk }) => {
+      test.skip(
+        packageName === 'nextjs-sdk-next-app' ||
+        packageName === 'gen1-next14-pages' ||
+        packageName === 'gen1-next15-app' ||
+        packageName === 'gen1-react' ||
```

Comment:
The reason `gen1-next14-pages`, `gen1-next15-app` and `gen1-remix` tests are skipped in `editing.spec.ts` is because those apps are not setup to work with visual editing. 

But `gen1-react` is, so it should be able to run this test. Did you try and it failed?

Reply:
It failed without the `isGen1VisualEditingTest` flag. Trying now with the flag set as `true`